backend/tests/integration-async/disambiguation/test_disambiguation_tasks.py

```diff
@@ -1418,3 +1418,16 @@ def assert_disambiguation_task():
         assert "Davis Gross, Brian" == literature_record_from_es_authors[0]["full_name"]
 
     retry_until_pass(assert_disambiguation_task, retry_interval=2)
+
+
+def test_disambiguation_when_empty_authors(inspire_app, clean_celery_session, enable_disambiguation):
+
+    literature_data = faker.record("lit", with_control_number=True)
+    literature_record = LiteratureRecord.create(literature_data)
+    db.session.commit()
+
+    def assert_disambiguation_task():
+        assert InspireSearch.get_record_data_from_es(literature_record)
+        assert disambiguate_authors(LiteratureRecord.get_uuid_from_pid_value(literature_record['control_number']), '1') is None
```

Comment:
It's wrong, you shouldn't trigger this task manually. It is triggered from receiver on db.session.commit() (so it means that after committing it will be executed automatically).

Reply:
sorry if I wasn't clear, the idea was to create a separate test to check the task itself and have also the integration test.

Reply:
Ah of course. Sorry I misunderstood. 👍  Thanks for clearing it up.